### ISSUE TITLE
Define multiple grouping aggregations as entities

### ISSUE THREAD

**Issue by joshuashort** [metabase/metabase#5483]
So, we have 'Segments' (for filters) and 'Metrics' (for views) to slice data, but how about something to pre-define 'grouping'? One example In my dataset is 'Hospital ID' and 'Hospital Name'. The name isn't guaranteed to be unique, but the ID is.. so it would be nice to provide a pre-defined group that includes ID and name, to provide what users expect while protecting them from the ugly realities of the data model. Does this make sense? Is it feasible? Otherwise, one has to group by ID and name to get the desired result..

**Comment by brunobergher** [comment]
I know it's been years, @joshuashort, but let me ask a question (since we're working on Dimensions right now).

Do you want to be able to define the merged ID + Name as a default grouping criterion, because name can't be unique? Have you considered moving the hospital names to a separate table and mapping the foreign key to it? IT should accomplish that exactly, nothing new needed (unless I'm missing something).

**Comment by joshuashort** [comment]
@brunobergher Yeah, it's been a while, and I'm somewhere else now; haven't used this software for a couple of years, but glad you're all pushing it forward :)

I think what I wanted to do was have grouping pre-sets that just happened to include those ID/Name fields -- the capability has other uses. Also, was trying to do all of this with minimal changes to the underlying schemas.

Sorry, I no longer have a use case :(

**Comment by brunobergher** [comment]
Alright, closing then.
